src/modules/seventv/global-emotes.js

```diff
@@ -0,0 +1,63 @@
+import watcher from '../../watcher.js';
+import settings from '../../settings.js';
+import AbstractEmotes from '../emotes/abstract-emotes.js';
+import {createEmote} from './utils.js';
+import {EmoteCategories, EmoteProviders, EmoteTypeFlags, SettingIds} from '../../constants.js';
+import {hasFlag} from '../../utils/flags.js';
+import formatMessage from '../../i18n/index.js';
+
+const category = {
+  id: EmoteCategories.SEVENTV_GLOBAL,
+  provider: EmoteProviders.SEVENTV,
+  displayName: formatMessage({defaultMessage: '7TV Global Emotes'}),
+};
+
+let eventSource;
+
+class SevenTVGlobalEmotes extends AbstractEmotes {
+  constructor() {
+    super();
+
+    watcher.on('channel.updated', () => this.updateGlobalEmotes());
```

Comment:
why do we update global emotes when a channel updates?

Reply:
Accidentally deleted the wrong line